TypeCobol.Test/Parser/CodeElements/CALL.ko.txt

```diff
@@ -7,8 +7,10 @@ Line 7[8,8] <27, Error, Syntax> - Syntax error : extraneous input 'g' expecting
 Line 7[1,4] <27, Error, Syntax> - Syntax error : mismatched input 'CALL' expecting {symbol, special register, keyword} RuleStack=codeElement>callStatement>cobolCallStatement>callReturningParameter>sharedStorageArea1>identifier>storageAreaReferenceOrConditionReference,  OffendingSymbol=[1,4:CALL]<CALL>
 Line 8[20,20] <27, Error, Syntax> - Syntax error : extraneous input 'y' expecting {separator, numeric literal, symbol, statement starting keyword, keyword, Formalized Comments elements} RuleStack=,  OffendingSymbol=[20,20:y]<UserDefinedWord>
 Line 11[17,21] <27, Error, Syntax> - Syntax error : missing {CONTENT, REFERENCE, VALUE} at 'MALUS' RuleStack=codeElement>callStatement>cobolCallStatement>callUsingParameters,  OffendingSymbol=[17,21:MALUS]<UserDefinedWord>
-Line 13[14,21] <27, Error, Syntax> - Syntax error : CALL .. USING: Illegal function identifier RuleStack=codeElement>callStatement>cobolCallStatement>callUsingParameters,  OffendingSymbol=[14,21:FUNCTION]<FUNCTION>
-Line 14[14,19] <27, Error, Syntax> - Syntax error : CALL .. USING: Illegal LENGTH OF in BY REFERENCE phrase RuleStack=codeElement>callStatement>cobolCallStatement>callUsingParameters,  OffendingSymbol=[14,19:LENGTH]<LENGTH>
+Line 13[23,28] <27, Error, Syntax> - Syntax error : CALL .. USING: Illegal function identifier OffendingSymbol=[23,28:RANDOM]<IntrinsicFunctionName>
+Line 14[14,19] <27, Error, Syntax> - Syntax error : CALL .. USING: Illegal LENGTH OF in BY REFERENCE phrase OffendingSymbol=[14,19:LENGTH]<LENGTH>
```

Comment:
Why is mentionned `in BY REFERENCE` for instruction on line 14?

Reply:
It is because the default way of passing data is BY REFERENCE, and when specified, it can be BY REFERENCE, BY VALUE, or BY CONTENT.